Objects/typeobject.c

```diff
@@ -3636,13 +3725,11 @@ PyType_FromMetaclass(PyTypeObject *metaclass, PyObject *module,
     }
 
     if (weaklistoffset) {
-        type->tp_weaklistoffset = weaklistoffset;
         if (PyDict_DelItemString((PyObject *)type->tp_dict, "__weaklistoffset__") < 0) {
```

Comment:
Out of curiosity; why are we doing this to the dict?

Reply:
They're descriptors of the metaclass (`type`), so AFAIK they're not necessary.